app/src/main/java/de/grobox/transportr/trips/detail/LegViewHolder.kt

```diff
@@ -194,9 +195,10 @@ internal class LegViewHolder(v: View, private val listener: LegClickListener, pr
         lineView.setWalk()
 
         // line color
-        fromCircle.setColorFilter(getColor(fromCircle.context, R.color.walking))
-        lineBar.setColorFilter(getColor(lineBar.context, R.color.walking))
-        toCircle.setColorFilter(getColor(toCircle.context, R.color.walking))
+        val color = MaterialColors.getColor(fromCircle.context, R.attr.colorSecondary, Color.TRANSPARENT)
+        fromCircle.setColorFilter(color)
+        lineBar.setColorFilter(color)
+        toCircle.setColorFilter(color)
```

Comment:
Not sure if I like having the walking color follow the theme.

Reply:
You are right, it leads to some readability issues. I'll reintroduce yellow.